docs/config/lua/config/font_rules.md

```diff
@@ -45,48 +45,53 @@ return {
 }
 ```
 
-Here's an example from my configuration file;
+Here's an example from a configuration file;
 
 ```lua
 local wezterm = require 'wezterm'
 
 -- A helper function for my fallback fonts
 function font_with_fallback(name, params)
-  local names = { name, 'Noto Color Emoji', 'JetBrains Mono' }
+  local names = { name, 'Noto Color Emoji', 'Symbols Nerd Font Mono' }
   return wezterm.font_with_fallback(names, params)
 end
 
 return {
-  font = font_with_fallback 'Operator Mono SSm Lig Medium',
+  font = font_with_fallback('FiraCode', {weight = 'Light'}),
+  font_size = 8.0,
   font_rules = {
     -- Select a fancy italic font for italic text
     {
       italic = true,
-      font = font_with_fallback 'Operator Mono SSm Lig Medium Italic',
+      intensity = 'Normal',
+      font = font_with_fallback('VictorMono', { weight = 'Regular', italic = true }),
     },
 
     -- Similarly, a fancy bold+italic font
     {
+      intensity = "Bold",
       italic = true,
-      intensity = 'Bold',
-      font = font_with_fallback 'Operator Mono SSm Lig Book Italic',
+      font = font_with_fallback('VictorMono', { weight = "DemiBold", style = 'Italic' }),
     },
 
     -- Make regular bold text a different color to make it stand out even more
     {
       intensity = 'Bold',
-      font = font_with_fallback(
-        'Operator Mono SSm Lig Bold',
-        { foreground = 'tomato' }
-      ),
+      font = font_with_fallback('FiraCode', { weight = 'Regular', italic = false, style = 'Oblique' }),
     },
 
     -- For half-intensity text, use a lighter weight font
     {
       intensity = 'Half',
-      font = font_with_fallback 'Operator Mono SSm Lig Light',
+      font = font_with_fallback('VictorMono', { weight = "ExtraLight", style = "Normal" }),
     },
   },
+    font_shaper = "Harfbuzz",
+    -- https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets
+    -- Victor Mono ss01-07 https://github.com/rubjo/victor-mono
+    harfbuzz_features = { 'cv13', 'cv18', 'cv31', 'kern', 'liga', 'clig', 'calt' },
+    -- ss04 is for victor as cv13 is 0 for fira that change $ with ss04
```

Comment:
This and the next line are unclear to me.

Reply:
Can be removed, as it is harfbuzz_features tuning and links to the fonts and some notes about the stylistic sets.
Becomes clear if you follow the Victor Mono and Fira Code links, and anyone new and wanting to use it would do that.

Reply:
Ideally, the notes can  be removed and harfbuzz_features accepted on a per rule basis, or merged.

Reply:
Nice to have an example that works with open fonts.
Both Fira Code and Victor Mono providing the Italics are OFL-1.1 licenses.

Specificing Symbols Nerd Font Mono changes icons for me, any need to have builtin at all by default?

Last commit is completely workable font example.
https://ibb.co/NW896gp